packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -124,11 +123,11 @@ export class Database {
     this.onDelete = config.onDelete || defaultOnDelete
   }
 
-  private collectionForPath = async (
-    filepath: string
-  ): Promise<Collection<true> | undefined> => {
+  private collectionForPath = async (filepath: string) => {
     const tinaSchema = await this.getSchema(this.level)
-    return tinaSchema.getCollectionByFullPath(filepath)
+    try {
+      return tinaSchema.getCollectionByFullPath(filepath)
+    } catch (e) {}
   }
```

Comment:
^ this change is to modify collectionForPath to never throw exceptions (just return undefined if unable to find the collection)